Floor Plan/projects/FloorPlan.json

```diff
@@ -1,16 +1,17 @@
 {
+  "ars_relationships" : [ ],
   "links" : [ {
+    "source" : "eventgenerator/ResidentGenerator",
+    "target" : "type/Resident"
+  }, {
     "source" : "type/Resident",
     "target" : "client/Resident"
-  }, {
-    "source" : "client/Resident",
-    "target" : "topic//my/response/topic"
   } ],
   "name" : "FloorPlan",
   "options" : {
-    "description" : "The FloorPlan tutorial demonstrates how to drop pins in coordinates on a 2D map or image. \n\nTo generated test data to power the client, click the play button next to the Resident Data Generator below and to the right.\n\nWith the client and data generator both running, you should see pins show up on both the first and second floor.\n\nTo see how the client was implemented, pause the client using the rounded square button on the Resident title bar and click around on the various widgets. Also take a look at the Resident Data Stream to see how data makes its way into the Client.",
+    "description" : "",
```

Comment:
the project lost the Project Description here, and no longer has the Description open in the panes below.
Do you need to change the project at all? 
If not, I'd revert this file

Reply:
Looks like that works just fine.